services/121-service/src/payments/fsp-integration/safaricom/services/safaricom.api.service.ts

```diff
@@ -35,6 +46,7 @@ export class SafaricomApiService {
 
     let errorMessage: string | undefined;
 
+    // ##TODO: unit test this part (and move to helper)
```

Comment:
Look at solution in IntersolveVisaApiService. If a similar solution can be applied here (with the customized http service), then consider refactoring that out into a Helper utils class, in the same way as the TokenValidationService.

Reply:
- I separated this to api helper service and wrote unit test
- it is not similar enough to the visa method to refactor into shared code
- given that I created the api helper service I also moved 'createTransferPayload' into there, which seems to make sense. Although you could then argue it's confusing why that public method has no unit test now.